### ISSUE TITLE
Empty paths vs null

### ISSUE THREAD

**Issue by scriptcoded** [robinvdvleuten/vuex-persistedstate#238]
- `vuex-persistedstate` version: `2.7.0`
- `node` version: `12.11.1`
- `npm` (or `yarn`) version: `npm 6.13.3`

Relevant code or config

```javascript
createPersistedState({
  paths: [
    // 'editor'
  ]
})(store)
```

**What you did:**

I had the `editor` Vuex module stored in localstorage. Since I temporarily disabled the module I commented it out in the persisted state config.

**What happened:**

I expected the library to not persist anything, but instead it persisted everything. 

**Reproduction sandbox:**

> This isn't really a bug. I'll leave this one out.

**Problem description:**

It makes it a bit confusing in a case like mine, especially if you haven't had a look at the documentation in a while. It took me a while to realize that it might be the store that was persisted.

**Suggested solution:**

The library should only persist the whole store if `paths == null`. The default should therefore also be `null`.

---

I'd be happy to raise a pull request if this is a feature that'd be appreciated.

**Comment by robinvdvleuten** [comment]
I agree with you that the `paths` option can be improved by allowing `null`. Could you create a PR for that?

**Comment by robinvdvleuten** [comment]
Solved by a34163fd75e48c08efe804e66896c8d59ffa2138
